### PR TITLE
HMOS DPreference Library

### PR DESCRIPTION
**Pull request by arpithakori**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +23,18 @@ Inspired by [Dozen Wang/DPreference](https://github.com/DozenWang/DPreference)
 ```
 2. For using dpreference in separate application using har file, add the har file in the entry/libs folder and add the dependencies in entry/build.gradle file.
 ```groovy
```
Comment (samuel-abhishek) [inline README.md:28]: Add the build and quality badges.

**Review by samuel-abhishek — CHANGES_REQUESTED** [review]
Please handle the comment

**Review by samuel-abhishek — APPROVED** [review]
Okay

**Review by sharathks84 — APPROVED** [review]
Release process Approved

**Review by Gleelakrishna — APPROVED** [review]
ok

**Pull request merged**